### PR TITLE
Show feed name inside the article

### LINKED ISSUE #3079: Show website name when opening article
This CSS rule remove the website name, but it seems that no rule display it elsewhere  :

```
.item.website span
{
    display: none;
}
```

Expected : We can see the website name with mobile view

### PR DESCRIPTION
**Pull request by Alkarex**
#fix https://github.com/FreshRSS/FreshRSS/issues/3079

### CODE REVIEW

**Comment by Alkarex** [comment]
Desktop (we could hide it, but it looks OK to me):
image

Mobile:

image

**Review by aledeg — APPROVED** [review]

File: app/views/index/normal.phtml [hunk 1]
```diff
@@ -72,20 +72,26 @@ $today = @strtotime('today');
...
 						foreach ($authors as $author):
 							echo $first ? _t('gen.short.by_author') . ' ' : '· ';
 							$first = false;
 				?>
 <em><a href="<?= _url('index', 'index', 'search', 'author:' . str_replace(' ', '+', htmlspecialchars_decode($author, ENT_QUOTES))) ?>"><?= $author ?></a></em>
 				<?php	endforeach; ?>
-				</div><?php
-					endif;
+				</div><?php endif; ?>
+				<div class="text"><?php
```
Comment (Alkarex) [inline app/views/index/normal.phtml:95]: I have added a new `<div class="text">...</div>` around the actual text of the article, to ease the layout (inlining) of our other information elements

File: p/themes/base-theme/template.rtl.css [hunk 2]
```diff
@@ -719,7 +719,7 @@ input[type="search"] {
 }
 
 .flux .item.title .author {
-	padding-right: 1rem;
+	padding-left: 1rem;
```
Comment (Alkarex) [inline p/themes/base-theme/template.rtl.css:722]: @Frenzie I am guessing it was a bug (there was no margin in RTL mode)

Comment (Frenzie) [reply p/themes/base-theme/template.rtl.css:722]: Possibly (can't check atm), but then it's better to do it with a control directive like `/*rtl:ignore*/` in the original so the fix won't be overwritten if you run `make rtl`.

**Review by marienfressinaud — APPROVED** [review]

**Comment by Alkarex** [comment]
A few more related comments https://github.com/FreshRSS/FreshRSS/issues/3079#issuecomment-650109551
Help welcome, otherwise I will try to find time in a couple of days

**Comment by marienfressinaud** [comment]
I didn't see them. I can't work on this today but the suggestions are good :+1:

**Comment by Alkarex** [comment]
Addressed https://github.com/FreshRSS/FreshRSS/issues/3079#issuecomment-650109551 but keeping the same design because authors can be something highly variable, as is typical for e.g. academic papers:
image

File: app/views/index/normal.phtml [hunk 3]
```diff
@@ -72,20 +72,29 @@ $today = @strtotime('today');
...
-						$first = true;
-						foreach ($authors as $author):
-							echo $first ? _t('gen.short.by_author') . ' ' : '· ';
-							$first = false;
-				?>
-<em><a href="<?= _url('index', 'index', 'search', 'author:' . str_replace(' ', '+', htmlspecialchars_decode($author, ENT_QUOTES))) ?>"><?= $author ?></a></em>
-				<?php	endforeach; ?>
-				</div><?php
-					endif;
+				<div class="subtitle">
```
Comment (Alkarex) [inline app/views/index/normal.phtml:75]: New subtitle section to ease styling

**Pull request merged**